docs/getting-started/running.mdx

```diff
@@ -671,7 +671,7 @@ Nuclei comes with reporting module support with the release of [v2.3.0](https://
 
 | **Platform** | GitHub                                                                                                                                                                                                                                                                                                            | GitLab                                                                                                                                                                                                                                                                                                            | Jira                                                                                                                                                                                                                                                                                                              | Markdown                                                                                                                                                                                                                                                                                                          | SARIF                                                                                                                                                                                                                                                                                                             | Elasticsearch                                                                                                                                                                                                                                                                                                     | Splunk HEC                                                                                                                                                                                                                                                                                                        |
```

Comment:
Could you please include the `duplicate-issue-check: true` in the "Nuclei Reporting" section as well?